src/Sylius/Bundle/CoreBundle/PriceHistory/Command/ClearPriceHistoryCommand.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\PriceHistory\Command;
```

Comment:
To make it clear that this is a command for the console:
```suggestion
namespace Sylius\Bundle\CoreBundle\PriceHistory\Cli\Command;
```

Reply:
https://github.com/Sylius/Sylius/pull/14904/commits/7a654469be7bd6ffb9a1d53cfcba21620055ed58